configure.ac

```diff
@@ -42,7 +42,7 @@ AC_ARG_ENABLE([compile-warnings],
        PICKY_CXXFLAGS="-Wextra -pedantic -Wno-long-long -Weffc++ -Wmissing-declarations"
        ;;
      error)
-       WARNING_CXXFLAGS="-Wall -Werror"
+       WARNING_CXXFLAGS="-Wall -Werror -Wno-c++17-extensions"
```

Comment:
Please update the link in the change description to "https://github.com/protocolbuffers/protobuf/issues/9181" directly, Also please leave a comment here with a TODO to remove this suppression once the issue is resolved.

Reply:
The only reason I didn't link the protobuf issue directly is due to the spam it creates in the protobuf issue each time I force push updates.

Do you still want it linked directly in the commit message?

Or perhaps I can add the link to this PR (ie the description above) to have a record of it after this is merged?

Reply:
I would prefer that there be a comment mentioning the issue/PR directly above the warning. I personally would not worry about spamming the issue -- hyperlinks and cross references exist for a reason.

Reply:
No problem - I added it to the commit message as well as to the comment in the file itself